lib/python/qmk/cli/doctor/check.py

```diff
@@ -129,20 +158,23 @@ def check_submodules():
     return CheckStatus.OK
 
 
-def is_executable(command):
+def is_executable(command, dict, warn=False):
     """Returns True if command exists and can be executed.
     """
     # Make sure the command is in the path.
     res = shutil.which(command)
     if res is None:
-        cli.log.error("{fg_red}Can't find %s in your path.", command)
+        if warn:
+            cli.log.warning("{fg_yellow}Can't find %s in your path.", command)
```

Comment:
Oh right I didn't thought about that case as it didn't appear to me because I have both the embecosm `riscv32-unknown-elf-gcc` and Debian packaged `riscv64-unknown-elf-gcc` toolchain installed on my machine. Instead of adding more logic to the check script I just removed the `riscv32-unknown-elf-gcc` check as we really only "support" the Debian path at the moment. So as long as other distribution lack the support for RISC-V toolchains this should be good. Hopefully the nomenclature on how the gcc executables are named will converge in the future. At the moment there is riscv64-elf-gcc, riscv64-unknown-elf-gcc among others.

Reply:
And on NixOS I have `riscv32-none-elf-gcc` (apparently they use `none` instead of `unknown` in such toolchains); their stock build did not work because of hardfloat/softfloat mismatch in libraries (multilib seems to be broken).